docs/topics/proc_running-the-containerless-mta-cli.adoc

```diff
@@ -0,0 +1,66 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-11-15
+:_mod-docs-content-type: PROCEDURE
+
+[id="running-the-containerless-mta-cli_{context}"]
+= Running the containerless CLI
+
+Starting from {ProductShortName} 7.2.0, you can perform an application analysis for Java applications by using the {ProductShortName} CLI that does not require installation of a container runtime. 
+
+[IMPORTANT]
+====
+In {ProductShortName} 7.2.0 and later, containerless CLI is a default mode. To enable container runtime usage, you must set the `--run-local` variable to `false`:
+
+----
+--run-local=false
+----
+====
+
+.Prerequisites
+
+* You have OpenJDK version 17 or later installed on your system.
+* You have Maven installed on your system.
```

Comment:
For maven commands, we find the binary via a `lookpath("mvn)`  to search for it in the PATH. We don't do this for the Java bin as we specifically want to use the one set in JAVA_HOME.

Reply:
So then that means that we should specify that the path to the `mvn` binary needs to be added to the PATH variable. @eemcmullan could you confirm? This is important, because Maven doesn't have an installer in Windows, so users would have to do that manually, so we should be pretty specific about it.

Reply:
@rromannissen  +1 I agree

Reply:
@rromannissen @eemcmullan  What about a Note then? Right under this one prereq? For example:

```
* You have Maven installed on your system.
+
IMPORTANT: You must also add the `mvn` binary file to the `PATH` variable.
```